common/icons/index.ts

```diff
@@ -86,7 +86,7 @@ export { default as singlePage } from './components/SinglePage';
 export { default as smartphone } from './components/Smartphone';
 export { default as snowflake } from './components/Snowflake';
 export { default as sort } from './components/Sort';
-export { default as soundcloud } from './components/Soundcloud';
+export { default as soundcloud } from './components/SoundCloud';
```

Comment:
nit: `default as soundCloud` maybe?

Reply:
I'm going to leave this for now – we don't interact other icon names, e.g. `youtube`.